core/db/migrate/20130319062004_change_orders_total_precision.rb

```diff
@@ -1,8 +1,8 @@
 class ChangeOrdersTotalPrecision < ActiveRecord::Migration
    def change
-    change_column :spree_orders, :item_total,  :decimal, :precision => 10, :scale => 2, :default => 0.0, :null => false
-    change_column :spree_orders, :total,  :decimal, :precision => 10, :scale => 2, :default => 0.0, :null => false
-    change_column :spree_orders, :adjustment_total,  :decimal, :precision => 10, :scale => 2, :default => 0.0, :null => false
-    change_column :spree_orders, :payment_total,  :decimal, :precision => 10, :scale => 2, :default => 0.0                                
+    change_column :spree_orders, :item_total,  :decimal, precision: 10, scale: 2, default: 0.0, null: false
+    change_column :spree_orders, :total,  :decimal, precision: 10, scale: 2, default: 0.0, null: false
+    change_column :spree_orders, :adjustment_total,  :decimal, precision: 10, scale: 2, default: 0.0, null: false
+    change_column :spree_orders, :payment_total,  :decimal, precision: 10, scale: 2, default: 0.0                                
```

Comment:
Line is too long. [129/120]<br>Unnecessary spacing detected.<br>Trailing whitespace detected.